pmg/static/resources/javascript/pmg.js

```diff
@@ -81,6 +81,14 @@ $(function() {
       $target.addClass('expanded');
       $btn.addClass('show-less');
       $btn.text('Show less');
+      ga('send', 'event', 'show-more', 'click');
```

Comment:
I wonder if it would make sense to have ``(...'recent-committee-meetings', 'show-more')`` and ``(... 'featured', 'show-more')``.